File: Assets/js/LocalizeText.js
#pragma strict
import UnityEngine.UI;
var originalText : String;
var text : Text;

function Awake() {
	text = GetComponent(Text);
	originalText = text.text;
}

function Start () {
	text.text = TextManager.GetText(originalText);
	Debug.Log(originalText + " -> " + text.text);
}

function Update () {

}

File: Assets/cs/LocalizeText.cs
namespace CSDEP
{
using UnityEngine;
using System;
using UnityEngine.UI;


public class LocalizeText:MonoBehaviour{
	
	public string originalText;
	public Text text;
	
	public void Awake() {
		text = GetComponent<Text>() as Text;
		originalText = text.text;
	}
	
	public void Start() {
		text.text = TextManager.GetText(originalText);
		Debug.Log(originalText + " -> " + text.text);
	}
	
	public void Update() {
	
	}
}
}
